README.md

```diff
@@ -0,0 +1,85 @@
+# openvpn
+
+This role installs OpenVPN, configures it as a server and can optionally create client certificates.
+
+
+## Requirements
+
+This role requires an apt based system.
+
+
+## Role Variables
+
+| Variable                           | Default / Mandatory                               | Description                                                                                                                                                     |
+|------------------------------------|---------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| openvpn_base_dir                   | `/etc/openvpn`                                    | Path where your OpenVPN config will be stored                                                                                                                   |
+| openvpn_key_dir                    | `/etc/openvpn/keys`                               | Path where your server private keys and CA will be stored                                                                                                       |
+| openvpn_port                       | `1194`                                            | The port you want OpenVPN to run on. If you have different ports on   different servers, I suggest you set the port in your inventory file.                     |
+| openvpn_server_hostname            | `{{inventory_hostname}}`                          | The server name to place in the client configuration file (if different from the `inventory_hostname`                                                           |
+| openvpn_proto                      | `udp`                                             | The protocol you want OpenVPN to use                                                                                                                            |
+| openvpn_dualstack                  | `true`                                            | Whether or not to use a dualstack (IPv4 + v6) socket                                                                                                            |
+| openvpn_config_file                | `openvpn_{{ openvpn\_proto }}_{{ openvpn_port }}` | The config file name you want to   use                                                                                                                          |
```

Comment:
Why is there a backslash between `openvpn` and `proto`?

Reply:
fixed